db/migrate/20221020122825_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Instead of using `user_id` as the `foreign-key` for your tables, I suggest you use `author_id` to avoid a conflict that could arise if you use user_id  with his index. 
You can do it as below:
 ```ruby
t.references :author, null: false, foreign_key: { to_table: 'users' } 
```
This will create author_id and a corresponding index